src/pages/RecipeEdit.js

```diff
@@ -3,13 +3,13 @@ import { useNavigate, useParams } from 'react-router-dom'
 import { Button, Form, FormGroup, Input, Label } from 'reactstrap'
 
 const RecipeEdit = ({recipes, updateRecipe}) => {
- const { id } = useParams()
+	const { id } = useParams()
```

Comment:
There are a lot of extra spaces in this file.  Can you select all in this file and do shift + tab.